src/share/classes/com/sun/javatest/regtest/tool/Tool.java

```diff
@@ -1125,9 +1101,11 @@ public void error(String msg) {
                 Tool.this.error(msg);
             }
         });
-        testManager.addTestFiles(testFileArgs);
-        testManager.addTestFileIds(testFileIdArgs);
-        testManager.addGroups(testGroupArgs);
+//        testManager.addTestFiles(testFileArgs);
```

Comment:
Did you intend to retain these commented lines?

Reply:
no, I thought I'd removed them all